index.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+ <meta charset="UTF-8">
+ <meta http-equiv="X-UA-Compatible" content="IE=edge">
+ <meta name="viewport" content="width=device-width, initial-scale=1.0">
+ <link rel="stylesheet" href="style.css">
+ <title>Portfolio</title>
+</head>
+<body>
+  <header class="header">
+    <nav class="navbar">
+      <a href="#" class="logo">
+        <h1 class="logo">My Logo</h1>
+      </a>
+      <div class="nave-toggle">
+        <img src="/image/Union.png" alt="" class="toggle-image">
+      </div>
+      <div class="menu">
+        <ul>
+          <li>Portfolio</li>
+          <li>About</li>
+          <li>Contact</li>
+        </ul>
+      </div> 
+    </nav>
+  </header>
+  <main>
+    <section class="page-1">
+      <img class="image image-2" src="/image/Ellipse11.png" alt="">
+      <img class="image image-3" src="/image/Rectangle7.png" alt="">
+      <img class="image image-4" src="/image/Rectangle4.png" alt="">
+      <img class="image image-5" src="/image/Ellipse3.png" alt="">
+      <img class="image image-7" src="/image/Ellipse13.png" alt="">
+      <img class="image image-8" src="/image/Ellipse12.png" alt="">
+      <img class="image image-9" src="/image/Rectangle5.png" alt="">
+      <img class="image image-1" src="/image/Rectangle6.png" alt="">
+                <!-- Backgound images -->
+      <img class="circle circle-1" src="/image/Ellipse8.png" alt="Circle 1">
+      <img class="circle circle-2" src="/image/Ellipse8.png" alt="Circle 2">
+      <img class="circle circle-3" src="/image/Ellipse8.png" alt="Circle 2">
```

Comment:
- [ ] As requested by the previous code reviewer, kindly remove these images and them as background images with CSS. As much as this is a core requirement, this also helps your webpage be more responsive and have good performance.